pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -208,6 +286,26 @@ func getAllDomainDisks(dom cli.VirDomain) ([]api.Disk, error) {
 	return devices.Disks, nil
 }
 
+func (l *LibvirtDomainManager) EFIBootable(secureBoot bool) bool {
```

Comment:
Hm, I think we had a misunderstanding here. I really think we should have our own structs for EFI and not merge it into the `LibvirtDomainManager`. I really only meant to only do the detection once when the domain manager gets created.

Reply:
Ah ok, mostly these `LibvirtDomainManager` methods (`EFICode`, `EFIVars` and `EFIBootable` should be moved to the `efiConfiguration` struct.

Reply:
Thank you for the comments, I've updated the PR, please take a second look.